.github/workflows/go.yml

```diff
@@ -0,0 +1,71 @@
+name: Go Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go_version: ['1.13', '1.14', '1.15']
+
+    steps:
+
+    - name: Set up Go version
+      uses: actions/setup-go@v1
```

Comment:
I noticed there is a newer version of ```setup-go```. Let's use v2

Reply:
Done.